src/index.html

```diff
@@ -1,11 +1,11 @@
-﻿﻿﻿<!DOCTYPE HTML>
+﻿﻿<!DOCTYPE HTML>
```

Comment:
HTML doesn't get linted. Anyway, I was just curious if you can see something suspicious.